src/content/deployment/subctl.en.md

```diff
@@ -46,6 +46,22 @@ to connect clusters to the Broker. This file contains the following details:
 |`--globalnet-cidr-range` `<string>` | Global CIDR supernet range for allocating GlobalCIDRs to each cluster (default "169.254.0.0/16")
 |`--ipsec-psk-from` `<string>`       | Import IPSEC PSK from existing Submariner broker file, like broker-info.subm (default "broker-info.subm)
 
+### export
+
+#### export service
+
+`subctl export service [flags] <name>` creates a `ServiceExport` resource for the given Service name. This makes the corresponding Service discoverable from other clusters in the Submariner deployment.
```

Comment:
JFYI, noticed that help section is not displaying the mandatory serviceName.
Raised the following issue for tracking - https://github.com/submariner-io/submariner-operator/issues/604